modules/local/mtx_to_seurat.nf

```diff
@@ -20,9 +20,9 @@ process MTX_TO_SEURAT {
     script:
     def aligner = params.aligner
     if (params.aligner in [ 'cellranger', 'cellrangerarc' ]) {
-        matrix   = "matrix.mtx.gz"
-        barcodes = "barcodes.tsv.gz"
-        features = "features.tsv.gz"
+        matrix   = "filtered_feature_bc_matrix/matrix.mtx.gz"
+        barcodes = "filtered_feature_bc_matrix/barcodes.tsv.gz"
+        features = "filtered_feature_bc_matrix/features.tsv.gz"
```

Comment:
This breaks the cellranger workflow. It seems one of the modules outputs these files in a subfolder, while the other does not.